### PR TITLE
Use location for logic

### PR DESCRIPTION
**Pull request by chrisbendel**

### CODE REVIEW

File: server/middleware.ts [hunk 1]
```diff
@@ -11,15 +12,22 @@ config.telefuncUrl = '/api/tf'
...
     app.all('/api/tf', async (req, res) => {
-        // debug geo headers
-        console.log(req.headers)
 
         const user = await getUserFromCookie(cookie.parse(req.headers.cookie || ''))
+
+        const userLocation = {
+            region: IS_PROD ? req.headers['cloudfront-viewer-country-region-name'] : 'Alabama',
+            country: IS_PROD ? req.headers['cloudfront-viewer-country'] : 'US'
+        }
```
Comment (chrisbendel) [inline server/middleware.ts:21]: @nathanstitt  i figured stubbing these in dev would be fine, let me know if there's a better alternative to this 🤔

File: server/user.ts [hunk 2]
```diff
@@ -15,22 +16,25 @@ export async function findUser(id: string): Promise<ChatbotUser | null> {
...
+    return yearsOld >= 16
 }
 
-export async function userEligibility(userId: string): Promise<UserEligibilityReply> {
-    if (!BETA_USERS.has(userId)) {
+export async function userEligibility(userId: string, userLocation: UserLocation): Promise<UserEligibilityReply> {
+    const isDelaware = userLocation.region === 'Delaware'
+    const isUsResident = userLocation.country === 'US'
+
+    if (!BETA_USERS.has(userId) || !isUsResident) {
```
Comment (chrisbendel) [inline server/user.ts:37]: We are now able to simplify and just not show the chatbot at all if they are outside of US 😄

File: lib/common.ts [hunk 3]
```diff
@@ -8,10 +8,6 @@ import { useQuery } from '@tanstack/react-query'
 
 config.telefuncUrl = '/api/tf'
 
-import type { FC } from 'react'
-
-export type { FC }
-
```
Comment (chrisbendel) [inline lib/common.ts:14]: This was causing some typing errors where all the component props were missing types, importing { FC } directly from react in the components works fine though

**Review by nathanstitt — APPROVED** [review]
looks good Chris, thanks

**Pull request merged**